neard/src/cli.rs

```diff
@@ -314,7 +314,7 @@ pub(super) struct TestnetCmd {
 
 impl TestnetCmd {
     pub(super) fn run(self, home_dir: &Path) {
-        nearcore::config::init_testnet_configs(
+        testlib::nearcore_test_utils::init_testnet_configs(
```

Comment:
So this I think is why we have to add `testlib` dependency -- we want to be able to run testnet via `neard`. 

It means that this functionality, isn't, strictly speaking, about nearcore internal testing -- this is a public API available to the users. So it seems to me that the current placement of the `GenesisExt` makes more sense. 

Why do we want to move it to the testlib?